External/FEXCore/Source/Interface/IR/Passes/ConstProp.cpp

```diff
@@ -198,6 +199,16 @@ class ConstProp final : public FEXCore::IR::Pass {
 
   std::unordered_map<uint64_t, OrderedNode*> ConstPool;
   std::map<OrderedNode*, uint64_t> AddressgenConsts;
+
+  // Pool inline constant generation. These are typically very small and pool efficiently.
+  tsl::robin_map<uint64_t, OrderedNode*> InlineConstantGen;
+  OrderedNode *CreateInlineConstant(IREmitter *IREmit, uint64_t Constant) {
+    if (InlineConstantGen[Constant]) {
```

Comment:
Maybe do something like:

```cpp
const auto iter = InlineConstantGen.find(Constant);
if (iter != InlineConstantGen.end()) {
  return iter->second;
}
```

So there's only one access into the map in this case